content/tutorial/02-sveltekit/01-concepts/04-server-side-rendering/README.md

```diff
@@ -2,4 +2,6 @@
 title: Server-side rendering
 ---
 
-TODO
+Server-side rendering (SSR) is the generation of the page contents on the server. It's what SvelteKit does by default when a user first visits your site.
+
+SSR is generally preferred for SEO (search engine optimization). While some search engines can index content that is dynamically generated on the client-side it may take longer even in these cases. It also tends to improve perceived performance and makes your app accessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)).
```

Comment:
I think we should put performance and resilience ahead of SEO, since a lot of people aren't concerned with SEO (but everyone should be concerned with the other two)

```suggestion
SSR tends to improve performance and makes your app accessible to users if JavaScript fails or is disabled (which happens [more often than you probably think](https://kryogenix.org/code/browser/everyonehasjs.html)). It is also preferred for search engine optimization (SEO) — while some search engines can index content that is rendered in the browser with JavaScript, it happens less frequently and reliably.
```